src/client/gui/gui_cmd.cpp

```diff
@@ -105,6 +106,16 @@ mp::ReturnCode cmd::GuiCmd::run(mp::ArgParser* parser)
     create_menu();
     tray_icon.show();
 
+    QFile first_run_file(QStandardPaths::writableLocation(QStandardPaths::AppDataLocation) + "/first_run");
+
+    if (!first_run_file.exists())
+    {
+        tray_icon.showMessage("Multipass System Tray Menu", "The Multipass system tray menu is now running",
```

Comment:
I'll sleep on this string, "system tray" apparently not a common enough term…

Reply:
Yeah, I struggled with the wording here...

Reply:
We said this will need to be platform specific though?

It's ["System menu"](https://support.apple.com/guide/mac-help/menu-bar-mchlp1446/mac) on macOS, ["Notification area"](https://en.wikipedia.org/wiki/Taskbar#Taskbar_elements) on Windows, ["App indicators"/"Status icons"/"System tray"](https://blogs.gnome.org/aday/2017/08/31/status-icons-and-gnome/) on Linux…

I'm thinking:
- "Multipass is in your System menu", "Click on the icon in the menu bar for available options"
- "Multipass is in your Notification area", "Click on the icon in the taskbar for available options"
- "Multipass is in your System tray", "Click on the status icon for available options"

I can't see a better way…

Reply:
> We said this will need to be platform specific though?

We originally said that because of what I thought at the time that Mac did not support clicking on the notification message and emitting a signal to invoke a slot (which is to open the default web browser to multipass.run).  However, that does work, so thus far, this code works across all platforms.  The only thing I've seen so far being platform specific is packaging changes to make this work.

That said, it's just strings and it's trivial to add platform specific string fetching via the platform interface.  I'll add that with the strings you suggest and we'll go from there.

Reply:
Ok, this is done.  It will of course break the other builds, but is trivial to fix that in full with the correct terminology.